### ISSUE TITLE
Emoji on message Error 

### ISSUE THREAD

**Issue by Demarky** [DiSkyOrg/DiSky#81]
I'm having trouble getting this code to work.

```
on message receive:
	if event-user is not a discord bot:
		if message is from guild:
			if discord id of event-channel is "974911306102607872":
				react to event-message with emoji "x"
```

After sending a message the following error appears

```
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! [Skript] Severe Error:
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Something went horribly wrong with Skript.
[18:28:38 ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[18:28:38 ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[18:28:38 ERROR]: #!#! Following plugins are probably related to this error in some way:
[18:28:38 ERROR]: #!#! DiSky
[18:28:38 ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[18:28:38 ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[18:28:38 ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[18:28:38 ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[18:28:38 ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Stack trace:
[18:28:38 ERROR]: #!#! java.lang.IllegalArgumentException: Emoji may not be null
[18:28:38 ERROR]: #!#!     at SK - DiSky.4.3.0.jar//net.dv8tion.jda.internal.utils.Checks.notNull(Checks.java:81)
[18:28:38 ERROR]: #!#!     at SK - DiSky.4.3.0.jar//net.dv8tion.jda.internal.entities.ReceivedMessage.addReaction(ReceivedMessage.java:172)
[18:28:38 ERROR]: #!#!     at SK - DiSky.4.3.0.jar//info.itsthesky.disky.api.emojis.Emote.addReaction(Emote.java:39)
[18:28:38 ERROR]: #!#!     at SK - DiSky.4.3.0.jar//info.itsthesky.disky.elements.sections.ReactSection.walk(ReactSection.java:58)
[18:28:38 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.EffectSectionEffect.walk(EffectSectionEffect.java:44)
[18:28:38 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[18:28:38 ERROR]: #!#!     at Skript.jar//ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
[18:28:38 ERROR]: #!#!     at SK - DiSky.4.3.0.jar//info.itsthesky.disky.api.events.DiSkyEvent.lambda$null$2(DiSkyEvent.java:143)
[18:28:38 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101)
[18:28:38 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[18:28:38 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1492)
[18:28:38 ERROR]: #!#!     at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:446)
[18:28:38 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1416)
[18:28:38 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1192)
[18:28:38 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305)
[18:28:38 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Version Information:
[18:28:38 ERROR]: #!#!   Skript: 2.6.3 (latest)
[18:28:38 ERROR]: #!#!     Flavor: skriptlang-github
[18:28:38 ERROR]: #!#!     Date: 00:55:22.847925631
[18:28:38 ERROR]: #!#!   Bukkit: 1.19.2-R0.1-SNAPSHOT
[18:28:38 ERROR]: #!#!   Minecraft: 1.19.2
[18:28:38 ERROR]: #!#!   Java: 17.0.4 (OpenJDK 64-Bit Server VM 17.0.4+8-Ubuntu-120.04)
[18:28:38 ERROR]: #!#!   OS: Linux aarch64 5.15.0-1013-oracle
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Server platform: Paper
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Current node: null
[18:28:38 ERROR]: #!#! Current item: react to event-types.message with emoji named "joy"
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Thread: Server thread
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! Language: english
[18:28:38 ERROR]: #!#! Link parse mode: DISABLED
[18:28:38 ERROR]: #!#!
[18:28:38 ERROR]: #!#! End of Error.
[18:28:38 ERROR]: #!#!

```

**Comment by SkyCraft78** [comment]
The error and the code seem not to be *related*, as the error points to another emoji that the one specified in the code.
This sort of error is most likely caused by wrong/unknown emojis. Try to restart your server and be sure the `/plugins/DiSky/emojis.json` file Is readable correctly.

**Comment by SkyCraft78** [comment]
Could be linked with #85

**Comment by SkyCraft78** [comment]
Is been linked with #85, and therefore has been fixed in v4.4.2. Thanks for the report!
